features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Applications/Details/Subscriptions.js

```diff
@@ -16,27 +16,104 @@
  * under the License.
  */
 
-import  React from 'react'
+import React from 'react'
 import Paper from 'material-ui/Paper';
 import Grid from 'material-ui/Grid';
 import Typography from 'material-ui/Typography';
+import Table, { TableCell, TableHead, TableRow, TableSortLabel, TableBody } from 'material-ui/Table';
+import PropTypes from 'prop-types';
+import SubscriptionTableData from './SubscriptionTableData'
+import Alert from '../../Shared/Alert';
 
-class Subscriptions extends React.Component{
-    constructor(props){
+import Subscription from '../../../data/Subscription';
+
+
+class Subscriptions extends React.Component {
+    constructor(props) {
         super(props);
+        this.state = {
+            subscriptions: null
+        }
+        this.handleSubscriptionDelete = this.handleSubscriptionDelete.bind(this);
     }
-    render(){
-        return (
-            <Paper>
-                <Grid container className="tab-grid" spacing={0} >
-                    <Grid item xs={12}>
-                        <Typography type="display1" gutterBottom >
-                            <span style={{fontSize: "50%"}}>Subscriptions page</span>
-                        </Typography>
+
+    componentDidMount() {
+        const client = new Subscription();
+        this.appId = this.props.match.params.applicationId;
```

Comment:
I think there is no usage of assigning `appId` to `this` instance as an instance variable here. Instead shall we take the applicationId as a constant here?
```javascript
const {applicationId} = this.props.match.params;
```